src/data/bench_locations.h

```diff
@@ -0,0 +1,6 @@
+static const struct BenchLocation sBenchPoints[] =
+{
+    [BENCH_HOME - 1]         = {MAP_GROUP(PALLET_TOWN), MAP_NUM(PALLET_TOWN), 3, 10}, //test location in Pallet Town
```

Comment:
Is there a reason we're not just zero-indexing the bench constants of subtracting 1 here?